test/ssl-tests/04-client_auth.conf.in

```diff
@@ -34,10 +34,12 @@ sub generate_tests() {
                 $caalert = "UnknownCA";
             }
             my $clihash;
+            my $clisigtype;
             my $clisigalgs;
             # TODO add TLSv1.3 versions
```

Comment:
I've been trying to put TLS1.3 TODO's in the form `TODO(TLS1.3)` so that it is easy to search the codebase and find them all.